lib/widget/download/download_grid.dart

```diff
@@ -86,22 +86,23 @@ class _DownloadGridState extends State<DownloadGrid> {
         type: PlutoColumnType.text(),
       ),
       PlutoColumn(
-          readOnly: true,
-          width: 90,
-          title: "Status",
-          field: "status",
-          type: PlutoColumnType.text()),
+        readOnly: true,
+        width: 115,
+        title: "Status",
+        field: "status",
+        type: PlutoColumnType.text(),
+      ),
       PlutoColumn(
         readOnly: true,
         enableSorting: false,
-        width: 128,
-        title: 'Transfer Rate',
```

Comment:
I would prefer the word transfer rate. Please roll this back as well as its width value,

Reply:
i used the `Speed` title, because it's a short name and doesn't take more space. anyway i revert title back to `Transfer Rate`